packages/suite/src/components/suite/Banners/index.tsx

```diff
@@ -0,0 +1,61 @@
+import * as React from 'react';
+import styled from 'styled-components';
+import { isDesktop } from '@suite-utils/env';
+import { useSelector } from '@suite-hooks';
+
+import OnlineStatus from './OnlineStatus';
+import UpdateBridge from './UpdateBridge';
+import UpdateFirmware from './UpdateFirmware';
+import NoBackup from './NoBackup';
+import FailedBackup from './FailedBackup';
+
+const Wrapper = styled.div`
+    display: flex;
+    flex-direction: column;
+    z-index: 3;
+`;
+
+const Banners = () => {
+    const transport = useSelector(state => state.suite.transport);
+    const online = useSelector(state => state.suite.online);
+    const device = useSelector(state => state.suite.device);
+
+    let Banner;
```

Comment:
we could add some default content for `Banner`

Reply:
That's probably not what we want. If there is no content, we don't want to show it at all. Would make sense if we wanted to avoid jumping of content when banner appears. But that would require some desining endeavour